src/main/java/screens/InfoScreen.java

```diff
@@ -0,0 +1,19 @@
+package screens;
+
+import javafx.scene.control.Label;
+import javafx.scene.layout.GridPane;
+
+/**
+ * Implementation of the window that handles showing information of the graph.
+ */
+public class InfoScreen extends GridPane {
+
+    /**
+     * Constructor.
+     */
+    public InfoScreen() {
+        getStyleClass().addAll("pane");
```

Comment:
Info screen and the graph screen should be edited once the program is actually capable of drawing.